packages/components/pie-chip/README.md

```diff
@@ -36,81 +27,13 @@ $ npm i @justeattakeaway/pie-chip
 $ yarn add @justeattakeaway/pie-chip
 ```
 
-For full information on using PIE components as part of an application, check out the [Getting Started Guide](https://github.com/justeattakeaway/pie/wiki/Getting-started-with-PIE-Web-Components).
-
-
-### Importing the component
-
-#### JavaScript
-```js
-// Default – for Native JS Applications, Vue, Angular, Svelte, etc.
-import { PieChip } from '@justeattakeaway/pie-chip';
-
-// If you don't need to reference the imported object, you can simply
-// import the module which registers the component as a custom element.
-import '@justeattakeaway/pie-chip';
-```
-
-#### React
-```js
-// React
-// For React, you will need to import our React-specific component build
-// which wraps the web component using ​@lit/react
-import { PieChip } from '@justeattakeaway/pie-chip/dist/react';
-```
-
-> [!NOTE]
-> When using the React version of the component, please make sure to also
-> include React as a [peer dependency](#peer-dependencies) in your project.
-
-
-## Peer Dependencies
+## Documentation
 
-> [!IMPORTANT]
-> When using `pie-chip`, you will also need to include a couple of dependencies to ensure the component renders as expected. See [the PIE Wiki](https://github.com/justeattakeaway/pie/wiki/Getting-started-with-PIE-Web-Components#expected-dependencies) for more information and how to include these in your application.
+Visit  [Link | PIE Design System](https://pie.design/components/chip/overview) to view more information on this component.
```

Comment:
I think "Link" isn't the most helpful text, and maybe we can give a hint about what information it includes, like usage, props and slots. That would also help anyone searching the page with Ctrl+F! 😄

Reply:
screams copy paste :P updated